chrome/content/zotero/itemPane.js

```diff
@@ -44,6 +45,23 @@ var ZoteroItemPane = new function() {
 		_notesList = document.getElementById('zotero-editpane-dynamic-notes');
 		_tagsBox = document.getElementById('zotero-editpane-tags');
 		_relatedBox = document.getElementById('zotero-editpane-related');
+		
+		var toggleReadButton = document.getElementById('zotero-feed-item-toggleRead-button');
+		var key = Zotero.Keys.getKeyForCommand('toggleRead');
+		toggleReadButton.setAttribute('tooltiptext', '(' + key + ')');
+
+		var addToLibraryButton = document.getElementById('zotero-feed-item-addTo-button');
+		key = Zotero.Keys.getKeyForCommand('saveToZotero');
+		if (key) {
+			// Add RLE mark in RTL mode to make shortcut render the right way
+			var tooltip = (Zotero.rtl ? ' \u202B' : '') + '('
+			+ (Zotero.isMac ? '⇧⌘' : Zotero.getString('general.keys.ctrlShift'))
+			+ key
+			+ ')';
+		}	
+		addToLibraryButton.setAttribute('tooltiptext', tooltip);
```

Comment:
The parentheses go after the regular tooltip (as in the existing "Zotero"/"Save to Zotero" buttons and the Firefox toolbar buttons).